ft_prepare_leadfield.m

```diff
@@ -185,6 +185,7 @@
 % check whether units are equal (NOTE: this was previously not required,
 % this check can be removed if the underlying bug is resolved. See
 % http://bugzilla.fieldtriptoolbox.org/show_bug.cgi?id=2387
+sourcemodel.unit = 'mm';
```

Comment:
what is the intended fix for this line? I suggest to remove it, because the units should not be silently added here, and there is no reason to assume that they are 'mm'.

Reply:
Good point. A local modification I forgot to remove.